message/association-release-request.go

```diff
@@ -139,3 +139,8 @@ func (m *AssociationReleaseRequest) MessageTypeName() string {
 func (m *AssociationReleaseRequest) SEID() uint64 {
 	return m.Header.seid()
 }
+
+// IsRequest returns true if the message is a request.
```

Comment:
```suggestion
// IsRequest reports whether the message is a request.
```

https://go.dev/doc/comment#func